service/discovery/k8s/network.go

```diff
@@ -0,0 +1,103 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+
+	"clouditor.io/clouditor/api/discovery"
+	"clouditor.io/clouditor/voc"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type k8sNetworkDiscovery struct{ k8sDiscovery }
+
+func NewKubernetesNetworkDiscovery(intf kubernetes.Interface) discovery.Discoverer {
+	return &k8sNetworkDiscovery{k8sDiscovery{intf}}
+}
+
+func (d *k8sNetworkDiscovery) Name() string {
+	return "Kubernetes Network"
+}
+
+func (d *k8sNetworkDiscovery) Description() string {
+	return "Discover Kubernetes network resources."
+}
+
+func (k k8sNetworkDiscovery) List() ([]voc.IsResource, error) {
```

Comment:
If the ingress controller is a load balancer, than the AccessRestriction and the url relationship is missing from the Ontology.

Reply:
Ingress is now a loadbalancer. Loadbalancer is now a network service